### DISCUSSION TITLE
BED_SCREWS_ADJUST in touch mode?

### DISCUSSION THREAD

**Discussion by Kerozen9844** [KlipperScreen/KlipperScreen#1229] [Q&A] [answered]
Hello,
I'm having a problem running BED_SCREWS_ADJUST on Klipperscreen.
This command is functional, but only from the console. There is no touch interface like with SCREWS_TILT_ADJUST (bed_level).
I need to run BED_SCREWS_ADJUST from Moblieraker on my phone....
Not very practical.
How can I change this?
Thank you so much

**Comment by alfrix** [comment, marked as answer]
there is no need for BED_SCREWS_ADJUST in klipperscreen because you are supposed to use the corner buttons on the bed_level panel to move the nozzle above the screw directly, no need to mess with an unnecessary wizard

for SCREWS_TILT_ADJUST there is a button in the bed_level panel if it was defined in the config
